vignettes/blueprintr.Rmd

```diff
@@ -40,88 +40,113 @@ cache_location <- tempdir()
 drake::clean(cache = drake::drake_cache(cache_location))
 ```
 
-blueprintr is a companion to [drake](https://github.com/ropensci/drake) that focuses on documenting and testing tabular data. Whereas drake manages the workflow execution, blueprintr defines a collection of steps that need to be run in a drake workflow.
+`blueprintr` is a framework for managing your data assets in a reproducible fashion. While it uses [drake](https://github.com/ropensci/drake) or [targets](https://cran.r-project.org/web/packages/targets/), it adds automated steps for tabular dataset documentation and testing. This allows researchers to create a replicable framework to prevent programming issues from affecting analysis results.
 
-# Basic Use
+## Installation 
 
-The first, and recommended, step is to attach blueprintr to your R session with `library()`.
+```{r setup, results= FALSE}
+# install.packages("remotes")
+# remotes::install_github("nyuglobalties/blueprintr")
 
-```{r setup}
 library(blueprintr)
 ```
 
-In a [drake project](https://books.ropensci.org/drake/projects.html), all packages that you want attached are declared in a `"packages.R"` file. This `library(blueprintr)` command should go there.
+## Designed Use of blueprintr
+`blueprintr` provides your data with guardrails typically found in software engineering workflows. 
+This allows you to test and document before deploying to production.
 
-blueprintr is built around "blueprints." Our first blueprint will be a blueprint for `mtcars`:
+The top level of the `blueprintr` workflow is a "blueprints" directory, consisting of `.R` files-- each a blueprint. 
```

Comment:
To mesh with the definition that follows, I'd say either: 

1. List both .R and .csv files OR
2. Change "each a blueprint" to "each a part of a blueprint" or something similar

Reply:
thx 4 catching this!